ui/components/app/detected-token/detected-token-aggregators/detected-token-aggregators.js

```diff
@@ -11,18 +12,22 @@ import {
   TYPOGRAPHY,
 } from '../../../../helpers/constants/design-system';
 
-const DetectedTokenAggregators = ({ aggregatorsList }) => {
-  const t = useContext(I18nContext);
-  const numOfHiddenAggregators = parseInt(aggregatorsList.length, 10) - 2;
+const DetectedTokenAggregators = ({ aggregators }) => {
+  const t = useI18nContext();
+  const numOfHiddenAggregators = parseInt(aggregators.length, 10) - 2;
```

Comment:
Can we make `2` a constant with a meaningful name?  When looking at this, I don't immediately understand what it represents.